src/app-layout/visual-refresh/drawers.tsx

```diff
@@ -300,7 +300,6 @@ function DesktopTriggers() {
             selected={hasSplitPanel && isSplitPanelOpen}
             ref={splitPanelRefs.toggle}
             highContrastHeader={headerVariant === 'high-contrast'}
-            testId="awsui-app-layout-trigger-slide-panel"
```

Comment:
👀 this may come up as a breaking change (even if undocumented). I will keep an eye how this gets released

Reply:
This was only added 2 weeks ago in [#2746](https://github.com/cloudscape-design/components/pull/2746), so should be relatively contained. But still a good call out.